tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py

```diff
@@ -557,7 +557,7 @@
         Field(
             item="65B",
             name='RECEIVE_SSI',
-            friendly_name="receives ssi",
+            friendly_name="Receives Disability Benefits: Supplemental Security Income Under Title XVI-SSI or Aged, Blind, and Disabled Under Title XVI-AABD ",
```

Comment:
@victoriaatraft one more little tweak, could we remove the extra space at the end of this?